jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/pom.xml

```diff
@@ -562,6 +562,7 @@
         <dependency>
           <groupId>org.wildfly.arquillian</groupId>
           <artifactId>wildfly-arquillian-container-remote</artifactId>
+          <version>2.1.0.Final</version>
```

Comment:
Same as with the other one, just change version in dependencyManagement few lines above, please.